modules/sagemaker/sagemaker-model-monitoring/requirements-dev.in

```diff
@@ -0,0 +1,12 @@
+awscli
```

Comment:
Is this duplicate of `requirements-dev.in`/`requirements-dev.txt` in the root of the repo?

Reply:
It was copied from `modules/sagemaker/sagemaker-endpoint/`, I didn't fully audit what was in it.

Reply:
I don't think we need to have `requirements-dev.txt` in our module folders. We ignore it during module installation, and also during our static checks. So I think we should delete these two files.